src/main/kotlin/io/polygon/kotlin/sdk/websocket/PolygonWebSocketClient.kt

```diff
@@ -20,14 +20,31 @@ import kotlinx.serialization.json.*
 private const val EVENT_TYPE_MESSAGE_KEY = "ev"
 
 /**
- * See https://polygon.io/sockets for details
+ * Feed is the data feed (e.g. Delayed, RealTime) which represents the server host.
  */
-enum class PolygonWebSocketCluster(internal vararg val pathComponents: String) {
+enum class Feed(val url: String) {
+    Delayed("delayed.polygon.io"),
+    RealTime("socket.polygon.io"),
+    Nasdaq("nasdaqfeed.polygon.io"),
+    PolyFeed("polyfeed.polygon.io"),
+    PolyFeedPlus("polyfeedplus.polygon.io"),
+    StarterFeed("starterfeed.polygon.io"),
+    LaunchpadFeed("launchpad.polygon.io")
+}
+
+/**
+ * Market is the type of market (e.g. Stocks, Crypto) used to connect to the server.
+ */
+enum class Market(val market: String) {
     Stocks("stocks"),
+    Options("options"),
     Forex("forex"),
     Crypto("crypto"),
-    Options("options"),
     Indices("indices"),
+    LaunchpadStocks("stocks"),
+    LaunchpadOptions("options"),
+    LaunchpadForex("forex"),
+    LaunchpadCrypto("crypto")
```

Comment:
I experimented with using a sealed class here instead of enum to support an "Other" option in this PR: https://github.com/polygon-io/client-jvm/pull/119

LMK what you think about the trade-offs there. Personally I really like the forward-compatibility that the sealed class supports even with the slight trade-off of how Java usages access these fields.

Reply:
Added a comment in the other thread https://github.com/polygon-io/client-jvm/pull/119#issuecomment-1662552909

Reply:
Fixed in #122